Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocAutoCompleteReferenceValue.jquery.css

```diff
@@ -1,142 +1,121 @@
+  z-index: var(--z-index-max);
 }
 
 .ac_results > div
 {
   width: 100%;
   height: 100%;
-  border: 1px solid Black;
-  background-color: White;
   box-sizing: border-box;
-  -moz-box-sizing: border-box;
-  -webkit-box-sizing: border-box;
+  padding: var(--remotion-themed-spacing);
+  background-color: var(--color-standard-background);
   overflow: auto;
 }
 
 .ac_results ul
 {
+  display: table;
   list-style-position: outside;
   list-style: none;
+  width: 100%;
   padding: 0;
   margin: 0;
-  width: 100%;
-  display: table;
 }
 
 .ac_results li
 {
   display: table-row;
-  cursor: default;
+  padding: var(--remotion-themed-spacing);
+  color: var(--color-standard-text);
+  cursor: pointer;
 }
 
 .ac_results li > div
 {
   display: table-cell;
-  padding-top: 0.1em;
-  padding-bottom: 0.1em;
-  padding-left: 0.2em;
-  padding-right: 0.2em;
-  line-height: 1.5em;
-  white-space: nowrap;
+  padding: var(--remotion-themed-spacing) var(--remotion-themed-spacing-small);
+  color: var(--color-standard-text);
   vertical-align: middle;
+  white-space: nowrap;
+  outline: none;
 }
 
 .ac_results li > div:first-child
 {
-  width: 16px;
+  width: var(--default-icon-size);
+  padding-left: var(--remotion-themed-spacing);
+  border-top-left-radius: var(--remotion-themed-border-radius);
+  border-bottom-left-radius: var(--remotion-themed-border-radius);
 }
 
-.ac_results li > div:first-child img
+.ac_results li > div:last-child
 {
-  vertical-align: 0.1em;
-  width: 16px;
-  height: 16px;
+  padding-right: var(--remotion-themed-spacing);
+  border-top-right-radius: var(--remotion-themed-border-radius);
+  border-bottom-right-radius: var(--remotion-themed-border-radius);
 }
 
-.ac_loading
+.ac_results li > div img
 {
-  background-image: url('../Image/sprite.svg#AutoCompleteReferenceValueRequestInProgressIndicator');
-  background-size: 16px 16px;
-  background-position: right center;
-  background-repeat: no-repeat;
+  display: block;
+  width: var(--default-icon-size);
+  height: var(--default-icon-size);
 }
 
-.ac_over.ac_disabled
+.ac_results li:hover > div
 {
-  text-decoration: none;
-}
-
-.ac_over.ac_disabled > div
-{
-  position: relative;
-}
-
-.ac_over.ac_disabled > div:only-child::after,
-.ac_over.ac_disabled > div + div::after
-{
-  position: absolute;
-  left: 2px;
-  top: 2px;
-  right: 2px;
-  bottom: 2px;
-  display: inline-block;
-  content: '';
-  outline: dotted black 1px;
-  outline: auto 5px -webkit-focus-ring-color;
+  background-color: var(--color-highlight);
+  text-decoration: underline;
 }
 
-.ac_over.ac_disabled > div + div::after
+.ac_results li.ac_disabled > div span
 {
-  left: -18px;
+  color: var(--color-disabled-text);
 }
 
-.ac_even,
-.ac_even.ac_over.ac_disabled
-
+.ac_results li.ac_disabled > div img
 {
-  background-color: White;
-  color: Black;
+  opacity: 0.4;
+  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40), progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
```

Comment:
![We don't do that here](https://c.tenor.com/fBvQV_5Lp6UAAAAd/we-dont-do-that-here-black-panther.gif)
```suggestion
  opacity: 0.5;
  filter: grayscale(100%);
```